spec/rails_helper.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../../config/environment', __FILE__)
+abort("The Rails environment is running in production mode!") if Rails.env.production?
+require 'rspec/rails'
+
+ActiveRecord::Migration.maintain_test_schema!
+
+RSpec.configure do |config|
+  config.fixture_path = "#{::Rails.root}/spec/fixtures"
+
+  config.use_transactional_fixtures = true
+
+
+  config.infer_spec_type_from_file_location!
+
+  config.filter_rails_from_backtrace!
+
+  Shoulda::Matchers.configure do |config|
```

Comment:
This block here should probably be outside of the `RSpec.configure` block.. Double check this. Also, fix the indentation here